packages/playground/website/src/components/error-report-modal/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from 'react';
+import Modal from '../modal';
+import { addFatalErrorListener, logger } from '@php-wasm/logger';
+import { Button, TextareaControl, TextControl } from '@wordpress/components';
+
+import css from './style.module.css';
+
+export function ErrorReportModal() {
+	const [hasError, setHasError] = useState(false);
+	const [text, setText] = useState('');
+	const [logs, setLogs] = useState('');
+	const [url, setUrl] = useState('');
+	const [submitted, setSubmitted] = useState(false);
+
+	useEffect(() => {
+		addFatalErrorListener(logger, (e) => {
+			setHasError(true);
+			setText('');
+			setLogs((e as CustomEvent).detail.logs.join(''));
+			setUrl(window.location.href);
+		});
+	}, []);
+
+	function onClose() {
+		setHasError(false);
+		setText('');
+		setLogs('');
+		setUrl('');
+		setSubmitted(false);
+	}
+
+	function onSubmit() {
+		const data = ['What happened?', text, 'Logs', logs, 'Url', url].join(
+			'\n\n'
+		);
+		console.log(data);
+		// TODO: send data to the server
+		setSubmitted(true);
+	}
+
+	return (
+		<Modal isOpen={hasError} onRequestClose={onClose}>
+			{submitted && (
+				<header className={css.errorReportModalHeader}>
+					<h2>Thank you for reporting the error</h2>
+					<p>
+						We will look into the issue and open an{' '}
+						<a href="https://github.com/WordPress/wordpress-playground/issues/">
+							issue on GitHub if needed
+						</a>
+						.
```

Comment:
This is still unclear – it links to GitHub issues as if the user was expected to go there and report the problem. Let's link to the report on wp.org slack and say there's no need to fill an additional issue.

Reply:
Ok, let me see what the API returns and then I can figure out a better message.